common/buildings/hfe_buildings.txt

```diff
@@ -410,4 +526,1210 @@ hfe_building_thoth_module_industry = {
 			}
 		}
 	}
-}
\ No newline at end of file
+	on_destroy = {
+		owner = {
+			change_variable = {
+				which = hfe_building_thoth_module_industry
+				value = 1
+			}
+		}
+	}
+}
+
+hfe_building_high_mechanicus_godforge = {
+	icon = building_nano_forge
+	base_cap_amount = 1
+	base_buildtime = @fetime
+	can_build = no
+	can_be_disabled = no
+	can_demolish = no
+	can_be_ruined = no
+	add_to_first_building_slot = yes
+	planetary_ftl_inhibitor = yes
+	category = manufacturing
+	
+	potential = {
+#		planet = {
+#			OR = {
+#				has_planet_flag = hfe_varelse_colony
+#				has_planet_flag = hfe_colony_beta
+#			}
+#		}
+		owner = {
+			OR = {
+				has_country_flag = hfe_varelse
+				has_country_flag = hfe_varelse_primitive_0
+				has_country_flag = high_mechanicus_retained
+				has_country_flag = hfe_varelse_fedora_governance
+				has_civic = hfe_civic_high_mechanicus
+				has_civic = hfe_civic_overthrown_mechanicus
+			}
+		}
+	}
+
+	allow = {
+		planet = {
+			OR = {
+				has_planet_flag = hfe_varelse_colony
+				has_planet_flag = hfe_colony_beta
+			}
+		}
+		owner = {
+			OR = {
+				has_country_flag = hfe_varelse_primitive_0
+				has_country_flag = high_mechanicus_retained
+				has_country_flag = hfe_varelse_fedora_governance
+			}
+		}
+#		owner = { is_country_type = primitive }
+	}
+
+	country_modifier = {
+		armies_alloys_upkeep_mult = -0.05
+		armies_alloys_cost_mult = -0.05
+	}
+
+	planet_modifier = {
+		armies_alloys_cost_mult = -1
+		armies_alloys_upkeep_mult = -1
+		armies_energy_upkeep_mult = -0.5
+		armies_consumer_goods_cost_mult = -0.9
+		planet_pop_assembly_add = 1
+	}
+	
+	triggered_planet_modifier = {
+		potential = {
+			owner = { is_country_type = primitive }
+			owner = {
+				OR = {
+					has_country_flag = hfe_varelse_primitive_0
+					has_country_flag = high_mechanicus_retained
+					has_country_flag = hfe_varelse_fedora_governance
+#					has_government = hfe_gov_high_mechanicus
+				}
+			}
+			NOT = {
+				owner = { has_ethic = ethic_gestalt_consciousness }
+			}
+		}
+		modifier = {
+			job_hfe_job_godforge_primitive_add = 1
+			job_primitive_laborer_add = 5
+			job_primitive_priest_2_add = 1
+			job_researcher_add = -4 # Negating the Relic stuff
+			job_soldier_add = -1
+		}
+	}
+
+	triggered_planet_modifier = {
+		potential = {
+			owner = { has_country_flag = hfe_varelse_primitive_0 }
+			owner = {
+				OR = {
+					has_civic = hfe_civic_high_mechanicus
+					is_spiritualist = yes
+				}
+				NOT = { has_civic = civic_barbaric_despoilers }
+			}
+		}
+		modifier = {
+			job_noble_add = 1
+		}
+	}
+
+	triggered_planet_modifier = {
+		potential = {
+			owner = { has_country_flag = hfe_varelse_primitive_0 }
+		}
+		modifier = {
+			planet_metallurgists_minerals_upkeep_mult = -0.5
+		}
+	}
+
+	triggered_planet_modifier = {
+		potential = {
+			owner = { is_ai = yes }
```

Comment:
Im sure ai can handle district upkeep @

Reply:
No, we cannot take the chance of having to go back in some time in the future. Why do you think Glavius has a Mod out for it?